src/js/elements/tutorial.js

```diff
@@ -10,56 +10,63 @@ export class Tutorial extends Observable {
         super();
         this.wrapperClass = '.w-slider-mask';
         this.slideClass = '.w-slide';
-        
+
         this.tutorials = [];
         this.curIdx = 0;
     }
-    
+
     _preloadImages = () => {
         this.tutorials.forEach(slide => {
             let img = new Image();
             img.src = slide.image;
         })
     }
-    
+
     prepareDomElements = () => {
         this.tutorialContainer = $('.tutorial');
         this.tutorialSlider = $('.tutorial__slider');
         this.rightButton = $('.right-arrow', this.tutorialSlider);
         this.leftButton = $('.left-arrow', this.tutorialSlider);
         this.rightButton.off('click')
         this.leftButton.off('click')
-        
+
         this.rightButton.on("click", () => this.nextSlide());
         this.leftButton.on("click", () => this.prevSlide());
-        
+
         this.tutorialPageNumber = $('.tutorial__slide-number', this.tutorialContainer);
+        this.updateFirstSlide();
     }
 
     createSlides = (tutorialArr) => {
         this.tutorials = tutorialArr || [];
         if (this.tutorials.length > 0) {
             this._preloadImages()
-            this.prepareDomElements(); 
+            this.prepareDomElements();
             this.updateSlideNumber()
- 
+
         }
     }
-    
+
+    updateFirstSlide = () =>{
+        this.displaySlide(this.curIdx);
```

Comment:
can you instead of creating a new method just put `this.displaySlide(this.curIdx)` into `prepareDomElements` the rest ist just for analytics and we don't need the first slide event. thanks.

Reply:
Fixed